processor/k8sattributesprocessor/README.md

```diff
@@ -272,10 +272,19 @@ k8sattributes/2:
       - k8s.node.name
       - k8s.pod.start_time
     labels:
-     # This label extraction rule takes the value 'app.kubernetes.io/component' label and maps it to the 'app.label.component' attribute which will be added to the associated resources
-     - tag_name: app.label.component
-       key: app.kubernetes.io/component
-       from: pod
+      # This label extraction rule takes the value 'app.kubernetes.io/component' label and maps it to the 'app.label.component' attribute which will be added to the associated resources
+      - tag_name: app.label.component
+        key: app.kubernetes.io/component
+        from: pod
+    operator_rules:
+      # Apply the operator rules - see https://github.com/open-telemetry/opentelemetry-operator#configure-resource-attributes
+      enabled: true
+      # Also translate the following labels to the specified resource attributes: 
+      # app.kubernetes.io/name    => service.name
```

Comment:
It is mentioned before that these should be standardized at some point: https://github.com/open-telemetry/opentelemetry-helm-charts/issues/905

Reply:
OK - I'll work with semconv before I continue with this PR

Reply:
I've created https://github.com/open-telemetry/semantic-conventions/pull/1756 for the spec part